### PR TITLE
Add fr to default languages

### PR DESCRIPTION
**Pull request by dkatzz**
### Description

Add french to the languages available in staging

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Removed the release notes section if the title is sufficient for the release notes description, or put more details in that section.
- [ ] Created unit and/or browser tests which fail without the change (if possible)
- [ ] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).
- [ ] Ensured PII wasn't added to any new logs, unless it was guarded by `isDevOrStaging`

### Issue(s) this completes

Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: server/conf/application.conf [hunk 1]
```diff
@@ -180,7 +180,7 @@ play.i18n {
   # adding translations for programs and applications.
   #
   # Alphabetize all languages besides English.
-  langs = [ "en-US", "am", "zh-TW", "ja", "ko", "lo", "om", "ru", "so", "es-US", "tl", "ti" , "vi" ]
+  langs = [ "en-US", "am", "zh-TW", "fr", "ja", "ko", "lo", "om", "ru", "so", "es-US", "tl", "ti" , "vi" ]
```
Comment (gwendolyngoetz) [inline server/conf/application.conf:183]: (question) Do we know if all CEs are setting the languages in their deploy config files? If they are not they will suddenly have French come online.

Comment (dkatzz) [reply server/conf/application.conf:183]: I've checked all the production instances and confirmed they configure specific languages I think it's probably helpful on staging / by default to show all the languages that have CiviForm static text translation, but I can send a message confirming? I had just seen russian and some other new languages were added, so adding french would make it so all languages we have support for are included in this list.

Comment (gwendolyngoetz) [reply server/conf/application.conf:183]: It's likely I asked the same thing last time, lol. If prod configs are good that's plenty. If there are new languages on staging sites I wouldn't bother email people about it.

Comment (dkatzz) [reply server/conf/application.conf:183]: Sounds good! Thanks!

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**